### PR TITLE
Count down timer

### PR DESCRIPTION
**Pull request by martijnarts**

### CODE REVIEW

File: src/views/creatTimeElement.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+'user strict';
+import { TIMER_OUT } from '../constants.js';
+import createDOMElement from '../utils/createDOMElement.js'
+
+const creatTimerElement = () => {
+    const setTimeOut = createDOMElement('span', { id: TIMER_OUT });
+    setTimeOut.innerText = 'timer';
+    setTimeOut.setAttribute('id', 'timer');
```
Comment (martijnarts) [inline src/views/creatTimeElement.js]: This ID is already set on line 6

File: src/views/creatTimeElement.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+'user strict';
+import { TIMER_OUT } from '../constants.js';
+import createDOMElement from '../utils/createDOMElement.js'
+
+const creatTimerElement = () => {
+    const setTimeOut = createDOMElement('span', { id: TIMER_OUT });
+    setTimeOut.innerText = 'timer';
```
Comment (martijnarts) [inline src/views/creatTimeElement.js]: This text disappears after 1 second, are you sure you want it?

File: src/handlers/setTimer.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    const time = setInterval(myTimer, 1000);
+    
+    function myTimer() {
+    document.querySelector('#timer').innerHTML = `${sec} Second Left`;
+    sec--;
+    if (sec == -1) {
+        clearInterval(time);
+        alert(`Time Out`);
+    }
+}
```
Comment (martijnarts) [inline src/handlers/setTimer.js]: These lines should be indented with four spaces

**Review by martijnarts — COMMENTED** [review]
@bachar78

File: src/views/creatTimeElement.js (hunk 2 above)
Comment (bachar78) [reply src/views/creatTimeElement.js]: Yes we are

**Pull request merged**